yaml_files/simphony_metadata.yml

```diff
@@ -418,5 +418,36 @@ CUDS_KEYS:
     definition: Multiphase model
 
   ELECTROSTATIC_MODEL:
-    parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CFD, CUBA.MOLECULAR_DYNAMICS, CUBA.DEM]
+
+  BASIS:
+    definition: Space basis vectors
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.VECTOR:
+      shape: (3, 3)
```

Comment:
This should be shape: 3 
the Basis is a list of three vectors (which in turn are of shape 3).